cloudevents/formats/json-format.md

```diff
@@ -143,7 +143,8 @@ If the implementation determines that the type of data is `Binary`, the value
 MUST be represented as a [JSON string][json-string] expression containing the
 [Base64][base64] encoded binary value, and use the member name `data_base64` to
 store it inside the JSON representation. If present, the `datacontenttype` MUST
-reflect the format of the original binary data.
+reflect the format of the original binary data. If the `datacontenttype` is
+unspecified, processing MUST NOT assume a default of `application/json`.
```

Comment:
Based on the call last week, what about:
- - -
If a `datacontenttype` value is not provided, a default value of `application/json` MUST NOT be assumed nor used as the value of the `datacontenttype` attribute in the serialized CloudeEvent.
- - -
Without explicitly stating that it shouldn't appear in the CE, it wasn't clear to me what "processing" we were referring to.

TBH I wasn't sure if the "be assumed" is even needed but perhaps you wanted to give guidance about some internal processing (which we probably can't control anyway). If not, then perhaps:
- - -
If a `datacontenttype` value is not provided, no assumptions can be made as to the format of the data and therefore the `datacontenttype` attribute MUST not be present in the resulting CloudEvent.
- - -

Reply:
Going with the latter - I like that.